releases/release-1.31/release-notes/maps/pr-124281-map.yaml

```
@@ -0,0 +1,3 @@
+pr: 124281
+releasenote:
+  text: allow parameter to be set along with proto file path
```

```suggestion
  text: Allowed parameter to be set along with proto file path.
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653203981